addons/crm/wizard/crm_lead_to_opportunity.py

```diff
@@ -77,7 +77,7 @@ def _compute_duplicated_lead_ids(self):
                 convert.lead_id.partner_id.email if convert.lead_id.partner_id.email else convert.lead_id.email_from,
                 include_lost=True).ids
 
-    @api.depends('action')
+    @api.depends('action', 'lead_id')
```

Comment:
don't needed (this fix works : https://github.com/odoo/odoo/pull/69287)

Reply:
No required for the fix, but is a valid fix. Compute depends on lead_id .